### PR TITLE
Contact Github community

### PR DESCRIPTION
**Pull request by Bhuvana-Priya**
Added another point to the "request help or submit feedback" for developers to contact Github community.

### CODE REVIEW

**Review by JasonAndrewWriter — APPROVED** [review]

**Review by JasonAndrewWriter — APPROVED** [review]

File: microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md [hunk 1]
```diff
@@ -30,10 +30,10 @@ An overview of the changes involved, as described in [Overview of Manifest V3 -
 
 To help you plan, consider the following schedule for Microsoft Partner Center and Microsoft Edge changes.
 
+We are currently in the process of updating our MV3 migration timelines.
 
-#### Revised timeline
+As of now, we will continue to follow Chromium’s MV3 timelines. While we fundamentally align with these timelines, we will continue to analyze the concerns raised by extension developers and explore the right path for Microsoft Edge Addons ecosystem as a whole. We will independently decide on MV3 migration timelines for Microsoft Edge Addons and share an update here once they are available.
```
Comment (ashishnirvana1) [inline microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md:35]: Revise this to as follows. 


We are aware that Chrome has revised the timelines for Manifest V2 sunset and as of now we will continue to follow Chromium's timelines. We continue to analyze the concerns raised by the extension developers and explore the optimal path for Microsoft Edge Add-ons ecosystem. We will independently decide on MV3 migration timelines for Microsoft Edge Addons and share an update here. Meanwhile, please refer to the [Chromium timelines ](https://developer.chrome.com/docs/extensions/mv3/mv2-sunset/) for planning your extension's migration.

File: microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md [hunk 2]
```diff
@@ -30,10 +30,10 @@ An overview of the changes involved, as described in [Overview of Manifest V3 -
...
 
-This timeline is subject to change. This article will be updated as the milestones draw nearer.  Chrome has revised the timeline for Manifest V2 sunset, and we are about to update this article to reflect that change.
```
Comment (ashishnirvana1) [inline microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md:36]: Please change the January 2023 and June 2023 timelines in the table to "TBD - see note above"

**Comment by Bhuvana-Priya** [comment]
> Please update the changes as mentioned in the comments and share the update.

Thank you @ashishnirvana1 , I have made the changes as per your comments.

File: microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md [hunk 3]
```diff
@@ -30,16 +30,16 @@ An overview of the changes involved, as described in [Overview of Manifest V3 -
...
 
 | Timeframe	| Microsoft Partner Center changes | Microsoft Edge changes |
 |--- |--- |--- |
-| July 2022	| Microsoft Partner Center will no longer accept new Manifest V2 extensions with visibility set as `Hidden` or `Public`. | No change. |
-| January 2023 | Microsoft Partner Center will no longer accept updates to existing Manifest V2 extensions. Developers can submit updates for migrating a V2 extension to V3. | Microsoft Edge stops running Manifest V2 extensions. Enterprises can allow Manifest V2 extensions to run on Microsoft Edge using Enterprise policies. |
-| June 2023 | No change. | Manifest V2 extensions will no longer function in Microsoft Edge even with the use of Enterprise policy. |
+| TBD - See note above	| Microsoft Partner Center will no longer accept new Manifest V2 extensions with visibility set as `Hidden` or `Public`. | No change. |
```
Comment (ashishnirvana1) [inline microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md:40]: July 2022 timeline has already passed and that work is done. Pls do not change that.

Comment (Bhuvana-Priya) [reply microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md]: This is updated. Thanks

**Review by captainbrosset — APPROVED** [review]

**Comment by captainbrosset** [comment]
I took a look and see that all review comments have been addressed. Merging now.

**Pull request merged**